modules/terrain/src/lib/parse-terrain.js

```diff
@@ -89,15 +109,49 @@ function getMartiniTileMesh(terrainImage, terrainOptions) {
       boundingBox: getMeshBoundingBox(attributes)
     },
     mode: 4, // TRIANGLES
-    indices: {value: triangles, size: 1},
+    indices: {value: Uint32Array.from(triangles), size: 1},
     attributes
   };
 }
 
+/**
+ * Get Martini generated vertices and triangles
+ *
+ * @param {number} meshMaxError threshold for simplifying mesh
+ * @param {number} width width of the input data
+ * @param {*} terrain elevation data
```

Comment:
You could probably make this type `number[] | Float32Array`. Or really it could be any other numeric typed array type as well, so you could add more type unions.